catalog/ui/catalog-ui-search/src/main/webapp/component/query-editor/query-editor.view.js

```diff
@@ -130,6 +131,7 @@ module.exports = Marionette.LayoutView.extend({
         this.$el.toggleClass('is-text', false);
         this.$el.toggleClass('is-basic', false);
         this.$el.toggleClass('is-advanced', false);
+        this.$el.toggleClass('is-custom', false);
```

Comment:
Similar to above comment, doesn't look like we use these classes.  It might be that we could refactor `query-editor` and `query-add` to use the same component.

Reply:
Removed unused classes; attempted to refactor to use the same component, but it wasn't trivial.  Since this is outside the scope of templates, I recommend we make a tech debt ticket for this change as it exists in master already.